README.md

```diff
@@ -54,6 +72,9 @@ Provides async methods on a persistent connection to a kafka broker (server).  T
 ##### BrokerRouter
 Provides metadata based routing of messages to the correct Kafka partition.  This class also manages the multiple KafkaConnections for each Kafka server returned by the broker section in the metadata response.  Routing logic is provided by the IPartitionSelector.
 
+##### ProtocolGateway
+Wrap BrokerRouter Hide from you the implementation and handles failure by refreshing the metadata.
```

Comment:
Needs a better explanation, something like - 
A convenience class that allows sending Kafka protocol messages easily, including error handling and metadata refresh on failure.